pkg/maintainer/spv/spv.go

```diff
@@ -0,0 +1,63 @@
+package spv
+
+import (
+	"context"
+	"time"
+
+	"github.com/ipfs/go-log/v2"
+
+	"github.com/keep-network/keep-core/pkg/bitcoin"
+)
+
+var logger = log.Logger("keep-maintainer-spv")
+
+func Initialize(
+	ctx context.Context,
+	config Config,
+	chain Chain,
+	btcChain bitcoin.Chain,
+) {
+	spvMaintainer := &spvMaintainer{}
+
+	go spvMaintainer.startControlLoop(ctx)
+}
+
+type spvMaintainer struct {
+	config   Config
+	chain    Chain
+	btcChain bitcoin.Chain
+}
+
+func (sm *spvMaintainer) startControlLoop(ctx context.Context) {
+	logger.Info("starting SPV maintainer")
+
+	defer func() {
+		logger.Info("stopping SPV maintainer")
+	}()
+
+	for {
```

Comment:
It is fine to leave it uninitialized but let's add some step that will block this loop (e.g. `<- ctx.Done()`) before the actual logic is implemented. If we build the binary and start the maintainer without specifying flags, we will enter an endless loop printing log from `maintainSpv` all the time.

Reply:
Good point. Done in f736a8b20ea87f50ada2d1befd55820a9c975a3b.
Added `<-ctx.Done()` until the implementation is provided.